Cargo.toml

```
@@ -24,11 +24,13 @@ paste = "1.0.14"
 serde = { version = "1", features = ["derive"] }
 serde_yaml = "0.9.25"
 thiserror = "1.0.48"
-unic-langid = { version = "0.9.0", features = ["serde"] }
+unic-langid = { version = "0.9.6", features = ["serde"] }
 unicode-segmentation = "1.6.0"
 unscanny = "0.1.0"
 url = { version = "2.4", features = ["serde"] }
-biblatex = { version = "0.10.0", optional = true }
+biblatex = { git = "https://github.com/DerDrodt/biblatex.git", branch = "lang", features = [
```

We can do this now I think.
```suggestion
biblatex = { git = "https://github.com/typst/biblatex.git", features = [
```